packages/context/src/genesis.test.ts

```diff
@@ -0,0 +1,10 @@
+import { describe, test, expect } from 'vitest';
+import Penumbra1Genesis from './penumbra-1-genesis.json';
```

Comment:
organizationally, shouldn't extension static live in the `public` folder?

additionally, it seems that the json file is missing some fields: it's **483,361** lines long, compared to the **483,366** lines in this [genesis file](https://github.com/prax-wallet/web/pull/163/files)?

```
- "height": "0" (top)
-  "epochIndex": "0" (bottom)
- "altGasPrices": [] (bottom)
```

Reply:
> organizationally, shouldn't extension static assets live in the public folder?

I don't have a strong opinion, but the context package seemed to be the easiest place as it sets up the services for the extension.

> it seems that the json file is missing some fields?

Oh strange 🤔 . I did a sync and logged the .toJson() of the genesis block. It seems like it excluded fields that had the default value (?). The result is the same, but not really sure why it did that. Can go with yours.